dotnet/Cucumber.Messages/NdjsonSerializer.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Io.Cucumber.Messages.Types;
+using System;
+using System.Text.Json;
+
+namespace Cucumber.Messages
+{
+    public class NdjsonSerializer
+    {
+        private static Lazy<JsonSerializerOptions> _jsonOptions = new Lazy<JsonSerializerOptions>(() =>
+        {
+            var jso = new JsonSerializerOptions();
```

Comment:
I'm guessing this one is meant to be jso, is there maybe a better name for this to avoid confusion over a misname? `options` perhaps?

Reply:
Adopted the use of the variable name 'options'.